.github/workflows/ci.yml

```diff
@@ -46,6 +48,12 @@ jobs:
         task:
           - name: Test
             tox: tests
+        exclude:
+          # Twisted and thus trial don't work on Windows with CPython 3.9
```

Comment:
Maybe link to a Trac ticket for Twisted... I am sure this will be fixed ... or maybe is already fixed and just waiting for a new release.

But maybe it is worth migrating towcrier tests to stdlib unit test. In this way we will not depend on twisted for running the tests.

Reply:
Trial really has a bunch of nice facilities compared to stdlib unittest.  There are other options, of course, like pytest and testtools, but you're going to run into exactly the same issues with anything that's not part of the stdlib itself.

As far as Twisted's 3.9 release goes, there's https://twistedmatrix.com/trac/ticket/10027 at least

Reply:
True. Thanks for the link.